bin/i18n/utils/pegasus_markdown.rb

```diff
@@ -4,7 +4,7 @@
 
 module I18n
   module Utils
-    module PegasusMarkdown
+    class PegasusMarkdown
       def self.write_to(file_path, header, markdown)
```

Comment:
1. Create a constant `TRANSLATABLE_HEADERS` (maybe you can think of a better name)
```suggestion
    class PegasusMarkdown
      TRANSLATABLE_HEADERS = %w[title].freeze
    
      def self.write_to(file_path, header, markdown)
```

2. Replace `'title'` with the constant in the method: `header.slice(*self::TRANSLATABLE_HEADERS)` 
https://github.com/code-dot-org/code-dot-org/blob/342820266b715f9838441567e04a6ad0b123b904/bin/i18n/utils/pegasus_markdown.rb#L25-L27

Reply:
Good call! I like this simplification!